docs/framework/whats-new/index.md

```diff
@@ -60,8 +60,7 @@ The .NET Framework 4.7.2 includes new features in the following areas:
 
 A continuing focus in the .NET Framework 4.7.2 is improved accessibility, which allows an application to provide an appropriate experience for users of Assistive Technology. For information on accessibility improvement in the .NET Framework 4.7.2, see [What's new in accessibility in the .NET Framework](whats-new-in-accessibility.md). 
 
-<a name="core-472" />
-#### Core
+#### <a name="core-472" />Core
```

Comment:
would entering a blank line fix this instead? What I don't like about having that as part of the section name is that the anchor becomes #a-namecore-472-core on the published topic and it's hard to link to this

Reply:
The link doesn't work, probably because it's too near to its target. I don't think anything can be done about that, though.

Reply:
@Thraka Please make note of such things in the master issue. They'll have to be addressed the day of the Markdig conversion.